src/components/Card.js

```diff
@@ -86,13 +95,7 @@ export const Card = ({ direction = 'ltr' }) => {
         >
           <QRCode
             className="qrcode"
-            style={
-              !portrait
-                ? direction === 'ltr'
-                  ? { paddingRight: '1em' }
-                  : { paddingLeft: '1em' }
-                : {}
-            }
+            style={!portrait ? checkDirectionAndSetPadding : {}}
```

Comment:
Since you don't need the second case:
```suggestion
            style={!portrait && checkDirectionAndSetPadding}
```

Reply:
You're right. Commit